doc/source/user_guide/io.rst

```diff
@@ -464,6 +464,15 @@ worth trying.
 
    os.remove("foo.csv")
 
+.. ipython:: python
```

Comment:
Could you maybe add a small comment before the python block like: 

Setting ``use_nullable_dtypes=True`` will result in nullable dtypes for every column.

Between line 465 and 467

Reply:
Sure, done!